config/jsdoc/api/plugins/api.js

```diff
@@ -148,7 +166,7 @@ exports.handlers = {
         // constructor from the docs.
         doclet._hideConstructor = true;
         includeAugments(doclet);
-      } else if (doclet.undocumented !== false && !doclet._hideConstructor && !(doclet.kind == 'typedef' && doclet.longname in types)) {
+      } else if (!doclet._hideConstructor && !(doclet.kind == 'typedef' && doclet.longname in types)) {
```

Comment:
This change was only for readability, because `doclet.undocumented` is either `true` or `undefined`, but never `false`.